src/modules/storage/BlockStorage.ts

```diff
@@ -63,30 +65,44 @@ export class BlockStorage extends Storages
                 (data.header.validators == undefined) ||
                 (data.header.validators._storage == undefined) ||
                 (data.header.merkle_root == undefined) ||
-                (data.header.signature == undefined)
+                (data.header.signature == undefined) ||
+                (data.txs == undefined)
         ) {
             if (callback != undefined)
                 callback("Parameter validation failed.");
             return;
         }
 
+        var tx_count: number = 0;
+        var enrollment_count: number = 0;
+
+        tx_count = data.txs.length;
+
+        if (data.enrollments)
+            enrollment_count = data.enrollments.length;
```

Comment:
```suggestion
        var tx_count: number = data.txs.length;
        var enrollment_count: number = 0;
        if (data.enrollments)
            enrollment_count = data.enrollments.length;
```

At the very least. And I believe you can do the same for enrollments ? Otherwise just use a ternary.

Reply:
use a ternary.